src/simulation/dynamics/gravityEffector/polyhedralGravityModel.cpp

```diff
@@ -145,7 +145,7 @@ PolyhedralGravityModel::computeField(const Eigen::Vector3d& position_planetFixed
 }
 
 double
-PolyhedralGravityModel::computePotentialEnergy(const Eigen::Vector3d& positionWrtPlanet_N) const
+PolyhedralGravityModel::computePotentialEnergy(const Eigen::Vector3d& position_planetFixed) const
```

Comment:
Just changed the position vector to pos_BP_P and added release notes.

I needed to squash because I committed my .venv by error (now I see what happened last time I tried squashing because I pulled before force pushing :/...). Squash has worked well this time but CI tests are not triggering?